specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1930,6 +2076,100 @@
         }
       },
       "description": "The backup key result, containing the backup blob."
+    },
+    "KeyPolicy": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The key policy id.",
+          "readOnly": true
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeActions"
+          },
+          "description": "Actions that will be performed by Key Vault over the lifetime of a key."
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyPolicyAttributes",
+          "description": "The key policy attributes."
+        }
+      },
+      "description": "Management policy for a key."
+    },
+    "LifetimeActions": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/LifetimeActionsTrigger",
+          "description": "The condition that will execute the action."
+        },
+        "action": {
+          "$ref": "#/definitions/LifetimeActionsType",
+          "description": "The action that will be executed."
+        }
+      },
+      "description": "Action and its trigger that will be performed by Key Vault over the lifetime of a key."
+    },
+    "LifetimeActionsTrigger": {
+      "properties": {
+        "timeAfterCreation": {
+          "type": "string",
+          "format": "string",
+          "minimum": 1,
+          "maximum": 99,
+          "description": "Days after creation to attempt rotate."
+        },
+        "timeBeforeExpiry": {
+          "type": "string",
+          "format": "string",
+          "description": "Days before expiry to attempt rotate."
+        }
+      },
+      "description": "A condition to be satisfied for an action to be executed."
+    },
+    "LifetimeActionsType": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the action.",
+          "enum": [
+            "Rotate",
```

Comment:
Please add descriptions for these enum values. Also, as noted above, they are typically camelCase, but that's not a critical issue.

Reply:
I noticed the case wasn't changed. These values are indeed acceptable to the service (i.e. if case-sensitive, are the correct case)? They differ in case than every other enum value in the swagger.

Reply:
Done